public/src/components/channelManagement/bannerTests/bannerTestVariantEditor.tsx

```diff
@@ -90,8 +102,41 @@ const BannerTestVariantEditor: React.FC<BannerTestVariantEditorProps> = ({
     onVariantChange({ ...variant, secondaryCta: updatedCta });
   };
 
+  function isBannerTemplate(s: string): s is BannerTemplate {
+    return Object.values(BannerTemplate).includes(s as BannerTemplate);
+  }
+
   return (
     <div className={classes.container}>
+      <div className={classes.sectionContainer}>
+        <Typography className={classes.sectionHeader} variant="h4">
+          Banner template
+        </Typography>
+        <RadioGroup
+          aria-label="Default"
+          name="default"
+          className={classes.templates}
+          value={variant.template}
+          onChange={(event, value): void => {
+            if (isBannerTemplate(value)) {
+              onVariantChange({
+                ...variant,
+                template: value,
+              });
+            }
+          }}
+        >
+          {Object.values(BannerTemplate).map(bannerTemplate => (
+            <FormControlLabel
+              key={bannerTemplate}
+              value={bannerTemplate}
+              control={<Radio disabled={!editMode} />}
+              label={bannerTemplate}
+            />
+          ))}
+        </RadioGroup>
+      </div>
```

Comment:
Maybe we could factor this into its own component?

Reply:
Good question, I'm undecided.
It would be neater here, but it's also a lot of boilerplate for a small component, e.g. it needs its own `classes`, we have to pass in the callback etc.
Also, if something like this is moved out, should the container/heading come with it, or remain in the parent?

Reply:
Yeah I hate that Material UI boilerplate 😕. I think if we wanted to 'prettify' the labels (i.e. ContributionsBanner -> Contributions, DigitalSubscriptionsBanner -> Subscriptions banner) it might be just enough logic to justify it. Good question! I've been leaving the header and container in the parent i.e

```jsx
      <div className={classes.buttonsSectionContainer}>
        <Typography className={classes.buttonsSectionHeader} variant="h4">
          Buttons
        </Typography>

        <VariantEditorButtonsEditor
          primaryCta={variant.cta}
          secondaryCta={variant.secondaryCta}
          updatePrimaryCta={updatePrimaryCta}
          updateSecondaryCta={updateSecondaryCta}
          isDisabled={!editMode}
          onValidationChange={onValidationChange}
        />
      </div>
```

This is from the same file. So Maybe it'd be something like

```jsx
      <div className={classes.sectionContainer}>
        <Typography className={classes.sectionHeader} variant="h4">
          Banner template
        </Typography>

        <BannerVariantEditorTemplateEditor
          template={variant.template}
          updateTemplate={updateTemplate}
        />
      </div>
```

I'm imagining we'd rename `buttonSectionContainer` and `buttonSectionHeader` to `sectionContainer` and `sectionHeader` as I think they can probably share styles (I think Hyuna would have tweaked the design slightly if she'd done it per variant)

Reply:
I had forgotten about prettyfying the labels! I think that's enough justification for a separate component

Reply:
I've moved it into a separate file (but had to refactor the models because of a weird webpack bug)